common/service-dialog-creation-using-dialog-editor.adoc

```diff
@@ -1,11 +1,11 @@
-{product-title} adds a new drag-and-drop service dialog editor. The editor, with its drag-and-drop feature, provides a visual representation of the components that comprise a service dialog - dialog tabs, boxes, and elements - to help you easily use them to design your dialog.
+{product-title} adds a new drag-and-drop service dialog editor to create service dialogs. The editor, with its drag-and-drop feature, provides a visual representation of the components that comprise a service dialog, that is dialog tabs, sections (previously referred as boxes), and elements to help you easily use them to design your dialog.
```

Comment:
* I do this all the time - but I would refrain from referring to a new feature as 'new' in the documentation itself, since technically, by the next release it's no longer new and not something you want to have to double back to remove down the road. Definitely a good feature synopsis for the release notes, though. 

* The second sentence might be best parceled out into two. Maybe something like "The editor, with its drag-and-drop feature, provides a visual representation of the components that comprise a service dialog. Easily design your dialog utlizing dialog tabs, secions (previously....) and elements.

Reply:
Ack, fixed.